### DISCUSSION TITLE
陽性判定（または発症）後即入院した接触確認アプリユーザーが陽性登録用処理番号を取得できる手段

### DISCUSSION THREAD

**Discussion by parasite2006** [cocoa-mhlw/cocoa#277] [Ideas]
## その機能リクエストは何らかの問題に関連しますか / Is your feature request related to a problem?

2021年7月1日以降の現行陽性登録手順で、接触確認アプリユーザーが本人確認用処理番号を取得する方法は
1) 保健所を通じて検査を受けた場合：陽性結果結果の電話連絡時に口頭で請求
2) 保健所を通じて検査を受け、無症状または軽症（酸素吸入、人工呼吸器などの呼吸管理がいらない）で陽性判定されて自宅/宿泊療養が決まり、健康観察データのスマホ入力用のMy HER-SYS IDを交付されている場合：健康観察データ入力時に「接触確認アプリ使用」と回答
https://www.mhlw.go.jp/content/10900000/000800366.pdf
3) 保健所を通じて検査を受け、無症状または軽症（酸素吸入、人工呼吸器などの呼吸管理がいらない）で陽性判定されて自宅/宿泊療養が決まり、スマホを使わず定電話（携帯電話）宛の自動架電（自動音声）に答えて健康観察データを入力する場合：健康観察データ入力時に「接触確認アプリ使用」と回答
https://www.mhlw.go.jp/stf/seisakunitsuite/bunya/kenkou_iryou/covid19_qa_kanrenkigyou_00009.html#Q6-5

の3通りが用意されているが、ユーザーが陽性判定（または発症）後に即入院した場合、陽性者が保健所に対し陽性登録の意思表示と処理番号の取得を行える方法が用意されておらず、陽性登録の盲点となっている。

これは以下の理由による。
1) 陽性判定（発症）後即入院した場合、自宅/宿泊療養の対象にならないので方法2と3が使えない。
2) 入院者のHER-SYSシステムへの登録と以後の転帰データの入力は医療機関側の仕事だが、処理番号発行の操作は保健所にしかできない。
3) 陽性登録に関しては医療機関と保健所の間に情報伝達経路がないため、ユーザー本人が保健所に連絡して処理番号を請求する以外に方法がない。
4) 特に本人が保健所を通さず自費で検査を受けた場合、保健所は結果連絡に関与せず、医療機関または本人からの連絡がない限り本人が陽性判定されたこと自体を知り得ない

## 解決策についてお書きください / Describe the solution you'd like

陽性判定（または発症）後に即入院したユーザー（当然ながら自宅/宿泊療養対象者より症状は重い）に対し、体への負担を極力かけずに確実に陽性登録の意思表示と処理番号の取得ができる手段を用意する。

## あなたが考える代替案についてご説明ください / Describe alternatives you've considered

例1) 医療機関がHER-SYSに感染症法に基づく発生届として入院者のデータを登録すると、保健所宛に通知メールが自動送信される。
（「HER-SYS簡易操作マニュアル 医療機関向け」のp.17
https://www.mhlw.go.jp/content/10900000/000777814.pdf ）
医療機関が発生届データの登録前に入院者本人に対して陽性登録の意思の有無を確認し、その結果をもとに通知メールの中に「（登録対象者が）接触確認アプリ使用」という情報を入れられるようにすれば、保健所は通知メールを見て処理番号発行操作を実行できる。

例2) 入院者が接触確認アプリユーザーで陽性登録を希望した場合、入院を受け入れ発生届を登録した医療機関に処理番号発行操作を実行する権限を与える。

例3) HER-SYSのシステム改修が必要な上2つの例の実現が難しい場合、せめて保健所内に接触確認アプリと陽性登録の担当者を決めてもらい、陽性登録希望者（陽性判定後即入院者に限らない）からの連絡はたらい回しなしで一元的にここにつながるようにしてもらう。

## その他 / Additional context

接触確認アプリ本体の機能やコードとは直接関係のない話題ではありますが、接触確認アプリユーザーが陽性判定（発症）後即入院というケースは現行の陽性登録システムの中では盲点となっており、この問題を少しでも広く知っていただきたくてあえて上げさせていただいた次第です。Discussionに移していただいて、広くご議論いただければ幸いに存じます。

**Comment by parasite2006** [comment]
続きです。
保健所は接触確認アプリの陽性登録以前にさまざまな業務の負担で手一杯になっている（データ入力一つとってもこの記事
https://www.buzzfeed.com/jp/naokoiwanaga/covid-19-system
のようにHER-SYS以外にも複数のデータベースへの入力を求められている）ことに加え、今年6月26日付（MY HER-SYS ID経由のスマホによる処理番号請求が始まる前）のiOS版COCOAのレビュー（下の画像2枚）
![IMG_0022+](https://user-images.githubusercontent.com/84975567/125935344-134c89d1-f5ca-42a3-82cf-76ae177ca389.PNG)
![IMG_0023+](https://user-images.githubusercontent.com/84975567/125935453-a564190a-84af-454b-ad8f-a4e9855689bc.PNG)
を見れば、（全部とは言えないまでも）保健所の中には健康観察にMY HER-SYS IDを使った健康観察データのスマホ入力を利用せずに電話をかけ続け（ひょっとするとHER-SYSへのデータ登録が滞っていてMY HER-SYS IDの確定を待つと時間がかかりすぎるためこうせざるを得なかったのかもしれない）、陽性登録用の処理番号（8桁）とMY HER-SYS ID（7桁）の違いも知らずに陽性登録希望者に対しMY HER-SYS IDを伝えてしまっている所があった（その結果接触確認アプリユーザーの陽性登録の意思が生かされないばかりか、アプリ自体に対する不信感まで招いている）ことがわかります。

こういうことなら、保健所の人手の関与を陽性登録の過程から極力はずすことが保健所とアプリユーザーの双方にとって、また信頼性が高い陽性登録システムを作る上で最善なのではないかと考えるに至りました。そこで以下を提案します。
陽性確認即入院となった接触確認アプリユーザーが陽性登録を希望する場合：入院を受け入れ発生届を登録した医療機関に処理番号発行操作を実行する権限を与える。（上の例2、HER-SYSの改修が必要）
保健所を通して検査を受けて陽性判定され、自宅/宿泊療養が決まった接触確認アプリユーザーが陽性登録を希望する場合：処理番号の請求経路をMY HER-SYS IDを使ったスマホからの請求
https://www.mhlw.go.jp/content/10900000/000800366.pdf
1本に絞り、保健所には
(1) 検査結果の電話連絡
(2) HER-SYSデータ登録とMY HER-SYS ID（6桁または7桁の番号）の確定
(3) MY HER-SYS IDを確定次第（療養場所の調整の完了を待たず）迅速に本人に通知し処理番号のスマホ請求案内チラシを同時配布
の3つだけやってもらえればいいとする
（HER-SYSのデータ登録完了と同時に登録された人の連絡先にMY HER-SYS IDがメールまたはSMSで自動的に届くようにできれば(1)と(2)の2つだけでよくなるのでなおよし。さらにMY HER-SYS IDの通知メッセージに接触確認アプリユーザー向けの陽性登録のお願い付きで処理番号のスマホ請求の案内
https://www.mhlw.go.jp/content/10900000/000800366.pdf
をリンクしておけば、チラシ配布の必要もなくなる）

**Reply by parasite2006** [reply]
厚労省が2021年9月2日付けで事務連絡https://www.mhlw.go.jp/content/000827144.pdf を出し、地元医師会に加えて診断医療機関にも自宅療養者の支援に加わるよう要請しています。その一環として、別紙https://www.mhlw.go.jp/content/000827145.pdf でMY HER-SYS医療機関向けマニュアルが改訂され、HER-SYSデータ（発生届）登録時に本人の連絡先（スマホ電話番号）を入力して「通知先」として指定し「MY HER-SYS URL通知」を「する」に設定すると、発生届の提出完了と同時にMY HER-SYS IDがSMSで自動通知されるようになりました。
陽性判定後即入院になったケースの場合は記載がありませんが、この手順なら発生届提出と同時にMY HER-SYS IDが自動通知されてくることになるでしょう。

**Reply by parasite2006** [reply]
2021年9月27日開催の厚労省の第53回アドバイザリーボード会議 https://www.mhlw.go.jp/stf/seisakunitsuite/bunya/0000121431_00294.html で、資料5の「陽性者へのきめ細かな情報の提供（接触確認アプリ（COCOA）関連」https://www.mhlw.go.jp/content/10900000/000836425.pdf として、陽性判明者のHER-SYSデータ登録完了と同時に接触確認アプリからの陽性登録に使える8桁の処理番号をもれなくSMSで自動通知する改修が提案されました（資料1ページ目）。SMSを受信した接触確認アプリユーザーがSMS内のリンクをクリックするだけで陽性登録画面に処理番号を入力した状態で接触確認アプリが起動し、あとは画面上の「登録する」ボタンを押すだけで即陽性登録が完了するようになります（資料2ページ目）。

この提案が通れば、陽性判明した接触確認アプリユーザーが自宅にいようが陽性判明後即入院していようが、HER-SYSデータ登録完了と同時に自分のスマホから陽性登録を実行できることになります。
（また陽性登録プロセスへの保健所の関与も、HER-SYSデータ登録の代行入力を除いてなくなります。検査結果の電話連絡は今後も行われるでしょうが、アプリの使用や陽性登録の意向の確認は不要になります）

ただし、この提案を接触確認アプリの陽性登録率の向上実現に結びつけるためには、
1) 陽性判明から陽性登録までのプロセスをあらかじめ接触確認アプリユーザーに周知すること、特にアプリ内に説明を表示して検査結果を待つ間に見ておけるようにすること
（処理番号通知のSMSを見落とされないように。これまでも処理番号がSMSで届くことを知らなかったユーザーが見落とした間に処理番号が期限切れすることがあった。）
2) 医療機関でのHER-SYSデータ登録プロセスの省力化・自動化
（データ登録負担の保健所への集中を避けると同時に、陽性検査結果確定からHER-SYSデータ登録完了までのタイムラグを短縮するため。実現には電子カルテメーカーやスキャナーメーカーにも協力を求め、電子カルテや紙の発生届からHER-SYSに登録する情報を抽出して登録用テンプレートに自動的に流し込むシステムを開発して発熱外来を開設する医療機関に配備する必要がある）
3) HER-SYSデータ登録件数に応じて医療機関にインセンティブを提供すること
（登録1件あたり保険点数何点、または補助金何円）

が必須だと考えます。

**Reply by parasite2006** [reply]
ご参考までに、4月の年度替わり以降厚労省のアドバイザリーボード会議で接触確認アプリとHER-SYSの運用が話題に上ったのは今回が3度目です。
1度目は2021年6月16日の第39回会議
https://www.mhlw.go.jp/stf/seisakunitsuite/bunya/0000121431_00256.html
の資料6「My HER-SYS 等を用いた接触確認アプリ処理番号発行（案）」
https://www.mhlw.go.jp/content/10900000/000793730.pdf
のp.4で厚労省が都道府県別の処理番号発行率と処理番号発行中陽性登録率を公表（調査期間は5月の最終週）。
<img width="951" alt="COCOARegistration(20210524-20210530)" src="https://user-images.githubusercontent.com/84975567/135784392-423858d7-23ed-4330-b33b-71b572e6b80b.png">

2度目は2021年7月28日の第45回会議
https://www.mhlw.go.jp/stf/seisakunitsuite/bunya/0000121431_00256.html#h2_free6
の資料3-3（西浦先生提出）
https://www.mhlw.go.jp/content/10900000/000812898.pdf
のp.84でOnset Date（発症日）からHER-SYS入力までとReport（検査結果の報告？）からHER-SYS入力までのタイムラグ日数を6月と7月で比較。
前者：mu=9.5日, var=95.2→mu=6.3日, var=35.9
後者：mu=6.9日, var=95.4→mu=3.3日, var=16.4

<img width="875" alt="HER-SYStimelag" src="https://user-images.githubusercontent.com/84975567/136635752-4b851471-22d5-48ea-9edb-86326e706fc4.png">

**Comment by kvaluation** [comment]
1. PCR検査の予約機能との連動
イングランドの接触確認アプリ NHS COVID-19 app で、接触通知を受信すると、アプリからPCR検査を予約できて、その検査で陽性だった場合、自動で陽性登録されるようです。
https://faq.covid19.nhs.uk/article/ka-01128/en-us

他のルートでの検査では、処理番号のようなものを使うようでした。

2. EN APIの新機能?
https://github.com/google/exposure-notifications-android/commit/610788eb3815d8ddb397332ba6f982308f7cde01

- Added an optional feature to allow users to upload a positive COVID test result when they don’t have a verification code from their health authority.
保健当局から検証コード（COCOA は処理番号）を受け取っていない場合でも、ユーザーが陽性の検査結果をアップロードできるようにするオプション機能を追加。
これは、本人だけでなく家族や入院先での本人のスマホを用いた代理入力もしやすいかもしれない。詳細は調べておりません。
